numpy/core/src/umath/loops_arithmetic.dispatch.c.src

```diff
@@ -35,14 +93,24 @@ simd_divide_by_scalar_contig_@sfx@(char **args, npy_intp len)
     const npyv_@sfx@x3 divisor = npyv_divisor_@sfx@(scalar);
 
     for (; len >= vstep; len -= vstep, src += vstep, dst += vstep) {
-        npyv_@sfx@ a = npyv_load_@sfx@(src);
-        npyv_@sfx@ c = npyv_divc_@sfx@(a, divisor);
+        npyv_@sfx@ c = simd_floor_divide_@sfx@(src, divisor, scalar);
```

Comment:
You will have to add a separate repeater for sign division, in order to achieve the example above https://github.com/numpy/numpy/pull/18766/files#r614468407